src/MudBlazor/Components/NumericField/MudNumericField.razor.cs

```diff
@@ -376,8 +396,11 @@ public T Max
         private T _step;
 
         /// <summary>
-        /// The increment added/subtracted by the spin buttons.
+        /// The amount added or subtraced when clicking the spin buttons.
```

Comment:
```suggestion
        /// The amount added or subtracted by the spin buttons.
```

Spin buttons can be used by the keyboard too

Reply:
How about: 

```
        /// <summary>
        /// The amount added or subtracted when changing values.
        /// </summary>
        /// <remarks>
        /// Defaults to <c>1</c>.  This affects changing values via spin buttons or the keyboard.
        /// </remarks>
```

Reply:
That works. I also like putting more in the summary:

```
        /// <summary>
        /// The amount added or subtracted when changing values via the spin buttons or keyboard.
        /// </summary>
        /// <remarks>
        /// Defaults to <c>1</c>. 
        /// </remarks>
```

Whichever you think's best